www/wp-content/object-cache.php

```diff
@@ -189,10 +189,6 @@ function delete($id, $group = 'default') {
 	}
 
 	function flush() {
-		// Don't flush if multi-blog.
-		if ( function_exists('is_site_admin') || defined('CUSTOM_USER_TABLE') && defined('CUSTOM_USER_META_TABLE') )
```

Comment:
@joshbetz so instead of removing this condition, what about changing it to being just `( defined( 'WPCOM_IS_VIP_ENV' ) && WPCOM_IS_VIP_ENV )`?